test/suite/wtbackup.py

```diff
@@ -343,12 +343,14 @@ def take_log_backup(self, bkup_c, backup_dir, orig_logs, log_cursor=None):
     # Optional arguments:
     #   consolidate: Add consolidate option to the cursor.
     #
-    def take_incr_backup(self, backup_incr_dir, id=0, consolidate=False):
-        self.assertTrue(id > 0 or self.bkup_id > 0)
-        if id == 0:
+    def take_incr_backup(self, backup_incr_dir, src_id=0, dest_id=0, consolidate=False):
+        self.assertTrue(dest_id > 0 or self.bkup_id > 0)
+        if src_id == 0 and dest_id == 0:
             id = self.bkup_id
-        # Open the backup data source for incremental backup.
-        config = 'incremental=(src_id="ID' +  str(id - 1) + '",this_id="ID' + str(id) + '"'
+            src_id = id - 1
+            dest_id =  id
+        # Open the backup data source for incremental backup.    
+        config = 'incremental=(src_id="ID' +  str(src_id) + '",this_id="ID' + str(dest_id) + '"'
```

Comment:
We can use an `fstring` here:
```
config = f'incremental=(src_id="ID{src_id}",this_id="ID'{dest_id)}"'
```

Reply:
[6933a2e](https://github.com/wiredtiger/wiredtiger/pull/9977/commits/6933a2ea61497bafac665a9f342e788504d8b62f)